code/datums/martial/sleeping_carp.dm

```diff
@@ -24,21 +24,22 @@
 	return FALSE
 
 /datum/martial_art/the_sleeping_carp/proc/strongPunch(mob/living/carbon/human/A, mob/living/carbon/human/D)
-	var/obj/item/bodypart/affecting = D.get_bodypart(ran_zone(A.zone_selected))
+	var/obj/item/bodypart/affecting = D.get_bodypart(ran_zone(A.zone_selected))//this var is so that the strong punch is always aiming for the body part the user is targeting and not trying to apply to the chest before deviating
```

Comment:
this is not codedocs

Reply:
So treat me like an idiot, is it the length or am I documenting this incorrectly?